irishlang.py

```diff
@@ -0,0 +1,63 @@
+import math
+import ast
+class irishspell:
+    def __init__(self):
+        self.file1 = open("irish.txt", "r", encoding = "UTF-16")

```

Comment:
Will you upload this irish.txt file as well so I can run this?

Reply:
[irish.txt](https://github.com/maddiekoehler/spellchecker-team5/files/7556842/irish.txt)